inventory-manager.lic

```diff
@@ -179,7 +181,41 @@ class InventoryManager
       end
     end
 
-    DRC.message "Saving eddy data for #{@active_character}!"
+    DRC.message("Saving eddy data for #{@active_character}!")
+    File.open(@item_db_path, 'w') { |file| file.write(@item_data.to_yaml) }
+    if using_sorter
+      start_script('sorter') unless Script.running?('sorter')
+    end
+  end
+
+  def add_storage_box_inv
+    if @item_data[@active_character]
+      @item_data[@active_character] = @item_data[@active_character].select { |line| !line.include?'caravan_box' }
+    else
+      @item_data[@active_character] = []
+    end
+
+    using_sorter = false
+    if Script.running?('sorter')
+      stop_script('sorter')
+      using_sorter = true
+    end
+
+    result = DRC.bput('rummage storage box', /In the storage box you see .*/i, 'What were')
+    tempString = result.slice!("You rummage through a storage box and see")
+    itemList = result.split(",")
+    itemList.each_with_index do |item, itr|
+      if itr < itemList.length-1
+        @item_data[@active_character] << item.lstrip.concat(' (caravan_box)')
+      else
+        item = item.split("and")
+                   .each do |item2| 
+                     @item_data[@active_character] << item2.strip.concat(" (caravan_box)").tr('.', '')
+                   end
+      end
+    end
+
+    DRC.message("Saving caravan box data for #{@active_character}!")
```

Comment:
This says 'caravan box', not 'storage box'. Are the two the same thing?

Reply:
Same thing. The storage box lives on a trader's caravan.